src/post-submission-actions/program-enrollment-action.ts

```diff
@@ -40,7 +39,11 @@ export const ProgramEnrollmentSubmissionAction: PostSubmissionAction = {
               title: t('enrollmentFailed', 'Enrollment failed'),
               kind: 'error',
               critical: false,
-              description: t('cannotEnrollPatientToProgram', 'This patient is already enrolled in the selected program'),            });
+              description: t(
+                'cannotEnrollPatientToProgram',
+                'This patient is already enrolled in the selected program',
+              ),
```

Comment:
```suggestion
              description: translateFrom(
              	 '@openmrs/esm-form-engine-lib',
                'cannotEnrollPatientToProgram',
                'This patient is already enrolled in the selected program',
              ),
```

Reply:
Hhmmm curious whether that would work. We may have to add the translations to the form engine app and instead:

```ts
description: translateFrom(
              	 '@openmrs/esm-form-engine-app',
                'cannotEnrollPatientToProgram',
                'This patient is already enrolled in the selected program',
              )

```

Reply:
> We may have to add the translations to the form engine app and instead:

Wait, why?

Reply:
Because it's not part of the import-map so at startup, the app-shell won't pick the resource bundles

Reply:
@ibacher , we are loading the translations for `@openmrs/form-engine-lib` dynamically as was implemented in this initialization PR for i18n. (specifically [this file](https://github.com/openmrs/openmrs-form-engine-lib/blob/03bb84e9f54a613da467d6f78f0feed68abea440/src/setupI18n.ts#L4))

Reply:
CC: @samuelmale

Reply:
FYI, `translateFrom` is basically using window.i18n 

Refer: https://github.com/openmrs/openmrs-esm-core/blob/35438c97f9a54cee71379a7760b050e71383678a/packages/framework/esm-translations/src/index.ts#L40

Reply:
In short, if we have the translation present in the `{lang}.json` file, then whenever a form is loaded, the {lang}.json is loaded dynamically and hence all the translations in the form-engine-lib is present in the window.i18n, under the namespace `@openmrs/form-engine-lib`
Thanks!

Reply:
Yes you're right @vasharma05. The i18n setup in the form-engine reconciles with the global one so theoretically it should work.

Reply:
That's right... I forgot about that.

Reply:
@vasharma05 Well, yes, but `translateFrom()` is also basically equivalent to:

```ts
import { i18n: { t } } from 'i18next';

// ...

t('key', { ns: moduleName });
```

Which would be functionally identical.

Reply:
Yes @ibacher, but when I tried this during the hackathon, it didn't work

The `i18next` was different from `window.i18n`, didn't get time to break down into it.